cinema/urls.py

```diff
@@ -1 +1,23 @@
-# write urls here
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+
+from cinema.views import (
+    GenreViewSet,
+    ActorViewSet,
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet
+)
+
+router = DefaultRouter()
+router.register("genres", GenreViewSet)
+router.register("actors", ActorViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+
+urlpatterns = [
+    path("", include(router.urls)),
+]
```

Comment:
Omit the router when there's no specified basepath. Utilize `urlpatterns = router.urls`  instead.